aws/sdk/build.gradle.kts

```diff
@@ -105,7 +105,12 @@ fun generateSmithyBuild(services: AwsServices): String {
                         "moduleDescription": "${service.moduleDescription}",
                         ${service.examplesUri(project)?.let { """"examples": "$it",""" } ?: ""}
                         "moduleRepository": "https://github.com/awslabs/aws-sdk-rust",
-                        "license": "Apache-2.0"
+                        "license": "Apache-2.0",
+                        "customizationConfig": {
+                            "awsSdk": {
+                                "integrationTestPath": "${project.projectDir.resolve("integration-tests")}"
+                            }
+                        }
```

Comment:
cool idea!! I ran into this doing some refactorings, this is a nice way to handle it